nemo_curator/__init__.py

```diff
@@ -37,7 +37,7 @@
 )
 from .utils.distributed_utils import get_client, get_network_interfaces
 
-# Dask will automatically convert the list score type
+# Dask automatically convert the list score type
 # to a string without this option.
```

Comment:
I don't think this change is needed. It should either be "Dask will automatically convert" or "Dask automatically converts".